exampleSite/config.toml

```diff
@@ -1,6 +1,6 @@
 baseURL = '/'
-languageCode = 'en'
-defaultContentLanguage = "en"
+languageCode = 'de'
+defaultContentLanguage = "de"
```

Comment:
please do not change the default language of the example site

Reply:
Sure!